docs/build/configure/triggers.md

```diff
@@ -250,17 +293,20 @@ When a trigger occurs, Viam sends a HTTP request to the URL you specified for th
 The request includes the following headers:
 
 <!-- prettier-ignore -->
-| Header Key | Description |
-| ---------- | ----------- |
-| `Part-ID` |  The part of the machine that triggered the request. |
-| `Robot-ID` | The machine that triggered the request. |
-| `Location-ID` | The location of the machine that triggered the request. |
-| `Org-ID` | The organization that triggered the request. |
-| `Component-Type` | The type of component for which data was ingested. Only for `part_data_ingested` triggers. |
-| `Component-Name` | The name of the component for which data was ingested. Only for `part_data_ingested` triggers. |
-| `Method-Name` | The name of the method from which data was ingested. Only for `part_data_ingested` triggers. |
-| `Min-Time-Received` | Indicates the earliest time a piece of data was received. |
-| `Max-Time-Received` | Indicates the latest time a piece of data was received. |
+| Header Key | Description | Trigger types |
+| ---------- | ----------- | ------------- |
+| `Org-Id` | The ID of the organization that triggered the request. | `part_data_ingested` |
+| `Organization-Name` | The name of the organization that triggered the request. | `part_online`, `part_offline` |
```

Comment:
@maxhorowitz also why is this so awkward? Shouldn't they all return org-id, org name, location id, location name, etc?

Reply:
TLDR is this is a bug - ticket is [here](https://viam.atlassian.net/browse/APP-5608) - is this something you'd like prio'd ?

Reply:
No all good. Was just wondering if it had been an oversight. No urgency from my end